package.json

```diff
@@ -25,5 +25,12 @@
   "description": "=========================== How to Use Grunt with Stanford Module Example ===========================",
   "main": "Gruntfile.js",
   "author": "",
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
Hey Dena,

Grunt and the other libraries are already listed as a dependency. They don't also need to be in the devDependencies for this.